client/transactions/filters/config.ts

```diff
@@ -106,7 +106,7 @@ export const filters: [ TransactionsFilterType, TransactionsFilterType ] = [
 
 /*eslint-disable max-len*/
 export const advancedFilters = {
-	/** translators: A sentence describing filters for Transactions. See screen shot for context: https://d.pr/i/NcGpwL */
+	/** translators: A sentence describing filters for Transactions. */
```

Comment:
Thanks for removing the stale screen shots! 🎉

Reply:
🥳  No problem